src/main/java/org/nebula/contrib/ngbatis/proxy/NebulaDaoBasicExt.java

```diff
@@ -299,42 +123,37 @@ public static String getCqlTpl() {
         return daoBasicTpl.get(getMethodName());
     }
 
-    /**
-     * 对输入接口的参数进行前置处理，转换成 {@link KV KV} 对象，为参数替换做准备
-     *
-     * @param record 待参数化对象
-     * @param fields 对象属性数组
-     * @param selective 是否排除空值
-     * @param prefix 参数前缀
-     * @return 属性占位符与属性值的键值对（双集合，相同下标成对）
-     */
-    public static KV recordToKV(Object record, Field[] fields, boolean selective, String prefix) {
-        KV kv = new KV();
-        for (Field field: fields) {
-            String name = null;
-            if( selective ) {
-                Object value = ReflectUtil.getValue(record, field);
-                if( value != null ) {
-                    name = field.getName();
-                }
-            } else {
-                name = field.getName();
-            }
-            if( name != null ) {
-                kv.columns.add( name );
-                // FIXME 使用 stmt 的方式，将实际值写入 nGQL 当中。
-                //  在找到 executeWithParameter 通过参数替换的方法之后修改成 pstmt 的形式
-                Object o = keyFormat( ReflectUtil.getValue( record, field ), name, true, prefix);
-                kv.valueNames.add( String.valueOf( o ) );
-            }
-        }
-        return kv;
+    public static Class<?> getClassFromStack( StackTraceElement stackTraceElement ) {
+        String className = stackTraceElement.getClassName();
+        Class<?> clazz = null;
+        try {
+            clazz = Class.forName(className);
+        } catch (ClassNotFoundException ignored) {} // 从方法栈中获得的 className 不会发生此异常
+        return clazz;
     }
 
-    public static void main(String[] args) {
-        String format = String.format("%s", null);
-        List<String> strings = Arrays.asList("dd", format, "ees");
-        System.out.println( String.join( ",", strings ));
+    public static MethodModel getMethodModel() {
+        StackTraceElement stackTraceElement = Thread.currentThread().getStackTrace()[2];
+        MethodModel methodModel = new MethodModel();
+        String methodName = stackTraceElement.getMethodName();
+        Class<?>  dao = getClassFromStack( stackTraceElement );
+
+        assert dao != null;

```

Comment:
suggest not use assert, except during the initialization.
how about print log info and throw exception?

Reply:
Good idea!